actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/feed/impl/FeedServiceImpl.java

```diff
@@ -85,7 +85,7 @@ public Response update(Feed feed, RequestContext context) {
     logger.info(context, "FeedServiceImpl:getRecordsByProperties method called : ");
     Response dbResponse =
         getCassandraInstance()
-            .getRecordsByProperties(
+            .getRecordsByPropertiesWithFiltering(
```

Comment:
Check this flow, it maybe that we are querying the feed table with user id. If not, we got to fix this by a script @reshmi-nair , @sknirmalkar89 .
The feeds are more or less an empty table - not many takers - so we needn't even write a script, just drop and recreate with some code adjustments. We can take it separately as a PR.

Reply:
@indrajra  we are querying the feed table with user id and category, the user_feed table primary key is id, and userid and category are indexed

Reply:
Yes, I realized so. But we can also fix it.